widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/exstensions/Merge.kt

```diff
@@ -26,7 +26,8 @@ import kotlin.reflect.full.primaryConstructor
  *     val b = MyDataClass(...)
  *     val c = a deepMerge b
  */
-internal inline infix fun <reified T : Any?> T.deepMerge(other: T): T {
+internal inline infix fun <reified T : Any?> T.deepMerge(other: T?): T {
+    if (other == null) return this
```

Comment:
Both changes were covered in original version, so you can rollout this changes

Reply:
Changed things back as they were and wrapped `Utils.getFullHybridTheme` to return empty `UiTheme` if merge resulted in `null`